### PR TITLE
add support for reading from embed.fs

### PR DESCRIPTION
**Pull request by preciselytord**

### CODE REVIEW

File: cfg.go [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+	Logger          Logger    // If set the logger will be used instead of standard out
+	MigrationsTable string    // Name of migrations table in database
+	Debug           bool      // Show debug info
+	DryRun          bool      // Perform all database operations but don't commit
+}
+
+func (c Config) filePath(fileName string) string {
+	if c.FS != nil {
+		return fileName
+	} else {
```
Comment (StefanNyman) [inline cfg.go]: unnecessary else

**Review by StefanNyman — APPROVED** [review]
looks good. just some stylistic nitpicks

File: cfg.go [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+		return filepath.Join(c.BaseDirectory, fileName)
+	}
+}
+
+func (c Config) fileContents(fp string) ([]byte, error) {
+	if c.FS != nil {
+		return c.FS.ReadFile(fp)
+	} else {
```
Comment (ndreas) [inline cfg.go]: Another unnecessary else

File: cfg.go [hunk 3]
```diff
@@ -1,12 +1,44 @@
...
+		return os.ReadFile(fp)
+	}
+}
+
+func (c Config) readDir() ([]fs.DirEntry, error) {
+	if c.FS != nil {
+		return c.FS.ReadDir(".")
+	} else {
```
Comment (ndreas) [inline cfg.go]: And another

**Review by ndreas — APPROVED** [review]
Nits, otherwise LGTM

**Pull request merged**